spec/struct.dd

```diff
@@ -95,8 +99,9 @@ $(H2 $(LNAME2 struct_layout, Struct Layout))
     attributes to describe an exact match. Using a $(DDSUBLINK spec/version, static-assert, Static Assert)
     to ensure the result is as expected.)
     $(LI Although the contents of the padding are often zero, do not rely on that.)
-    $(LI Avoid using empty structs when interfacing with C and C++ code.)
-    $(LI Avoid using empty structs as parameters or arguments to variadic functions.)
+    $(LI Avoid using Empty Structs when interfacing with C++ code that declares them with `extern "C"`.
+    Add a one byte dummy field to those structs.)
```

Comment:
Confusing. It's irrelevant if the C++ code declares then as `extern (C)` or not. And on D side, you can declare the `struct` as `extern (D)` or `extern (C++)` to accomplish the same - no need for the dummy field.

Reply:
You're right that empty C++ structs have sizeof 1 even for `extern "C++"` structs.

Reply:
I removed the line about `extern "C"`.